src/gui/pointcloud/qgspointcloudrendererpropertieswidget.cpp

```diff
@@ -191,6 +224,17 @@ void QgsPointCloudRendererPropertiesWidget::syncToLayer( QgsMapLayer *layer )
     {
       mLabels->setChecked( mLayer->renderer()->showLabels() );
       mLabelOptions->setTextFormat( mLayer->renderer()->labelTextFormat() );
+      mZoomOutOptions->setCurrentIndex( static_cast<int>( mLayer->renderer()->zoomOutBehavior() ) );
```

Comment:
Assuming the combo box index always matches the enum int value is fragile -- use this approach instead:

```suggestion
      mZoomOutOptions->setCurrentIndex( mZoomOutOptions->findData( QVariant::fromValue( mLayer->renderer()->zoomOutBehavior() ) ) );
```